### PR TITLE
Warm up and others

### PR DESCRIPTION
**Pull request by KrzysiekKnigawka**
Here are basically two major changes:

1. Add warm up:
  - set warm up time
  - set warm up desired eps
  - warm up runs with linear ramp up load profile. 25% of warm up time will be ramping up linearly to set eps
2. Add queue live results support. 

And some minor changes.

### CODE REVIEW

File: ServerlessBenchmark/LoadProfiles/LinearWithRumpUp.cs [hunk 1]
```diff
@@ -16,16 +16,22 @@ public class LinearWithRumpUp : TriggerTestLoadProfile
         private double _rampTimePercentage = 0.25;
 
         /// <summary>
-        /// First 10% of time use for ramp up and last 10% for ramp down
+        /// First 25% of time use for ramp up and last 25% for ramp down
         /// </summary>
         /// <param name="loadDuration"></param>
         /// <param name="eps"></param>
-        public LinearWithRumpUp(TimeSpan loadDuration, int eps) : base(loadDuration)
+        public LinearWithRumpUp(TimeSpan loadDuration, int eps, bool rampDown = true) : base(loadDuration)
```
Comment (paulbatum) [inline ServerlessBenchmark/LoadProfiles/LinearWithRumpUp.cs]: Typo: should be LinearWithRampUp

File: SampleUsages/Program.cs [hunk 2]
```diff
@@ -24,37 +24,37 @@ public new static void Main(string[] args)
 
         #region LambdaTests
         [Command]
-        public void S3Test(string functionName, string blobPath, string srcBucket, string targetBucket, string loadProfile, int eps = 0, bool repeat = false, int durationMinutes = 0)
+        public void S3Test(string functionName, string blobPath, string srcBucket, string targetBucket, string loadProfile, int eps = 0, bool repeat = false, int durationMinutes = 0, int warmUpTimeInMinutes = 3)
```
Comment (paulbatum) [inline SampleUsages/Program.cs]: I'm not sure if the default value of this new param should be 3 minutes. It should probably be 0 like durationMinutes. Same comment applies to other methods below.

File: ServerlessBenchmark/Constants.cs [hunk 3]
```diff
@@ -26,6 +26,8 @@ public static class Constants
         public const int HttpTriggerTimeoutMilliseconds = 300 * 1000;
 
         public const int LoadCoolDownTimeout = 180 * 1000;
+
+        public const int DefaultWarmUpTimeInSeconds = 60 * 3;
```
Comment (paulbatum) [inline ServerlessBenchmark/Constants.cs]: Where is this constant used? Sorry if I missed it...

**Comment by KrzysiekKnigawka** [comment]
Thanks @paulbatum I've addressed your comments.

**Pull request merged**